app/org/sagebionetworks/bridge/models/studies/PasswordPolicy.java

```diff
@@ -0,0 +1,85 @@
+package org.sagebionetworks.bridge.models.studies;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The policies that govern password creation. New studies will be created with the default password 
+ * policy, which requires 8 characters, one numeric, one symbol and one upper- and lower-case 
+ * character. However, this policy can be adjusted for legacy studies which were created with much 
+ * more lax constraints.
+ */
+public final class PasswordPolicy {
```

Comment:
We have to for backwards compatibility. Through the SDK, I have to set this to something lower for the existing studies. However we don't need to give researchers a UI to do this. The default for studies going forward will be higher than before.